b2sdk/utils.py

```diff
@@ -366,6 +366,24 @@ class B2TraceMetaAbstract(DefaultTraceAbstractMeta):
     pass
 
 
+class ConcurrentUsedAuthTokenGuard(object):
+    def __init__(self, lock_dict, lock_key):
```

Comment:
I think the caller should be responsible for getting the lock object from the dict, not the context manager

Reply:
@ppolewicz fixed in https://github.com/reef-technologies/b2-sdk-python/pull/53/commits/505e615aed937a5d23d896eb9d6d577577f3da3e